src/bindings/python/test.py

```diff
@@ -50,7 +51,43 @@
 
                 print("  %s %12s%12f" % (prefix, candidate['plate'], candidate['confidence']))
 
+finally:
+    if alpr:
+        alpr.unload()
+
+
+try:
+    alpr = Alpr(options.country, options.config, options.runtime_data)
```

Comment:
I don't think the cv option should be related to reading a stream or not. Because you could read from a stream and encode the video to byte array.It would still use opencv,  which is what we are doing at the moment.